exercises/concept/card-games/lists_test.py

```diff
@@ -60,9 +60,9 @@ def test_empty(self):
         want = []
 
         self.assertEqual(concatenate_rounds(rounds_1, rounds_2),
```

Comment:
There should be a newline after `assertEqual(` and indentation should be one level (4 spaces)